cmd/exchanger/main_test.go

```diff
@@ -0,0 +1,17 @@
+package main
```

Comment:
Generally when testing we want to "blackbox" test (https://www.imperva.com/learn/application-security/black-box-testing/)

to do this, in go, we typically call the package the name _test. So this test file should be in main_test. The nice thing about this too is it tells the compiler that it is ia test package and not to include it in our binary.